HelpSource/Classes/Function.schelp

```diff
@@ -30,6 +30,13 @@ code::
 ((lambda (x) x) 1) // Lisp
 ::
 
+NOTE:: 
+In case you already know how to define your own classes, you came accross the usage
+of teletype::^:: within methods to return values back to the caller. Please be aware
+that usage of teletype::^:: in functions not part of classes is not implemented yet
```

Comment:
@telephon Thank you for the feedback. I hope than my new commit is now complete for a merge. In case further corrections are needed, feel free to add them. Unfortunately I will have only a limited access to my system in the next weeks and that`s why can not estimate when I can add further changes.